### PR TITLE
fix: add location query variable

### LINKED ISSUE #5504: [CAL-364] Event Type clone defaults location to Cal Video
When you clone an event type it doesn't clone the location for some reason and always defaults to Cal Video, when it should ideally carry over the location data as well.

_And even after I manually set a cloned event type location to Zoom, it reverts to cal video after saving :\ After going back in to edit the new event and setting it to zoom again, it seems to stick_

> From [SyncLinear.com](https://synclinear.com) | [CAL-364](https://linear.app/calcom/issue/CAL-364/event-type-clone-defaults-location-to-cal-video)

### PR DESCRIPTION
**Pull request by Udit-takkar**
Fixes: #5504 

https://user-images.githubusercontent.com/53316345/202000736-136f0895-8c60-445f-a0f9-6ee64cec5246.mov

### CODE REVIEW

**Review by sean-brydon — APPROVED** [review]
LGTM - Ship it 🚀

File: packages/prisma/zod/custom/eventtype.ts [hunk 1]
```diff
@@ -9,8 +9,9 @@ export const createEventTypeInput = _EventTypeModel
     teamId: true,
     schedulingType: true,
     hidden: true,
+    locations: true
   })
-  .partial({ hidden: true })
+  .partial({ hidden: true,locations: true})
```
Comment (alannnc) [inline packages/prisma/zod/custom/eventtype.ts]:
```suggestion
  .partial({ hidden: true, locations: true })
```

**Review by alannnc — CHANGES_REQUESTED** [review]
Currently not working for location with links working on a fix

**Review by alannnc — APPROVED** [review]
Looks good for now, however this can be improved sending eventTypeId on the modal form, reading it's content on the trpc call and just duplicate data from DB instead of URL every parameter @Udit-takkar

**Pull request merged**